travis/test-spark-autologging.sh

```diff
@@ -16,7 +16,7 @@ popd
 TEMPDIR=$(mktemp -d)
 pushd $TEMPDIR
 wget https://archive.apache.org/dist/spark/spark-3.0.0-preview/spark-3.0.0-preview-bin-hadoop2.7.tgz -O /tmp/spark.tgz
-tar -xvf /tmp/spark.tgz
+tar -xf /tmp/spark.tgz
```

Comment:
Don't use verbose flag when extracting the tgz (it needlessly prints out all the extracted files & increases build log length)

Reply:
Thank you!